.watchmanconfig

```diff
@@ -0,0 +1,9 @@
+{
+  "ignore_dirs": [
+    "node_modules",
+    "react-native/node_modules",
+    "packages",
+    "integration-tests",
+    "tests"
+  ]
```

Comment:
Listing these files makes the watch more performant.
We would need to remove these if we want to start reacting to changes in these directories.
Side note: Unfortunately this config doesn't understand a `**/node_modules/**` glob 😞